lms/static/sass/course/_info.scss

```diff
@@ -56,6 +56,95 @@ div.info-wrapper {
       font-style: normal;
     }
 
+    div.upgrade-banner {
+      // This banner uses the Pattern Library's defined variables
+      background-color: $white;
+      border: 1px solid $uxpl-gray-background;
+      @include border-left(0px);
+      width: 100%;
+      display: table;
+
+      .green-border {
```

Comment:
We shouldn't include colors in class names, because other Open edX sites might use a different theme than ours. It would be good to name the class semantically, and ideally it would be a new class that could be added to the Pattern Library.